sensei-lms.php

```diff
@@ -86,7 +86,8 @@ function is_sensei_activating( $activating_plugin, $plugin ) {
 	require SENSEI_LMS_PLUGIN_PATH . 'vendor/woocommerce/action-scheduler/action-scheduler.php';
 }
 
-require_once dirname( __FILE__ ) . '/includes/class-sensei-dependency-checker.php';
+require_once __DIR__ . '/includes/class-sensei-dependency-checker.php';
+require_once __DIR__ . '/includes/class-sensei-pro-upsell.php';
```

Comment:
There is an effort for moving new internal classes (internal because they are not meant for use by other developers) to the `includes/internal` folder. Probably the `Sensei\Internal\Services` namespace would be a good place for this. Can you please move it there and add the `@internal` tag to the class? 🙏 

The `require_once` won't be needed afterwards because whose classes are loaded by the composer autoloader.